src/components/Board.js

```diff
@@ -5,16 +5,33 @@ import PropTypes from 'prop-types';
 
 
 const generateSquareComponents = (squares, onClickCallback) => {
-  // Complete this for Wave 1
+  // console.log(onClickCallback)
+ // Complete this for Wave 1
   // squares is a 2D Array, but 
   //  you need to return a 1D array
   //  of square components
 
+  let newArr = [];
+  for(let i = 0; i < squares.length; i++)
+  {
+      newArr = newArr.concat(squares[i]);
+  }
+
+  let squaresArr = [];
+  for(let i = 0; i < newArr.length; i++)
+  {
+      const currentSquare = newArr[i];
+      squaresArr.push ( <Square value={currentSquare.value} id={currentSquare} onClickCallback={onClickCallback}/>)
```

Comment:
1.  I just want to note that the propType for `id` is `Number`.  
2.  You're missing the `key` prop.

```suggestion
      squaresArr.push ( <Square key={ currentSquare.id } value={currentSquare.value} id={currentSquare.id} onClickCallback={onClickCallback}/>)
```